RobotCrawler/src/main/java/parser/BookDataGetter.java

```diff
@@ -0,0 +1,20 @@
+package parser;
+
+/**
+ * Created by ehsan on 25.08.16.
```

Comment:
Line about this would be good. Nothing fancy, just a line. It's a public interface and you already created JavaDocs.